index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
```

Comment:
`main` file of the application should be `index.js`.  But this is the code represent server is setup. Move this code inside `server.js` file and invoke that function from here. 

refer this https://github.com/Hdonge/nodejs-app/blob/master/app/index.js